qa-ibd.planx-pla.net/manifests/hatchery/hatchery.json

```diff
@@ -26,7 +26,23 @@
     },
     "containers": [
       {
-        "target-port": 8787,
+        "target-port": 8885,
+        "cpu-limit": "1.0",
+        "memory-limit": "1024Mi",
+        "name": "R Studio",
```

Comment:
why are we adding this one?

Reply:
oh, I added the old ones because we want all 6 options: 2 old versions, 2 new versions, 2 new versions with pre-installed packages, as described in the ticket https://ctds-planx.atlassian.net/browse/PXP-8416 😊